tests/test_dataset.py

```diff
@@ -517,14 +490,14 @@ def test_open_dataset_without_num_channels_in_properties() -> None:
         assert data["dataLayers"][0].get("num_channels") is None
 
     ds = Dataset(TESTOUTPUT_DIR / "old_wk_dataset")
-    assert ds.properties.data_layers["color"].num_channels == 1
-    ds.properties._export_as_json()
+    assert ds.get_layer("color").num_channels == 1
+    ds._export_as_json()
 
     with open(
         TESTOUTPUT_DIR / "old_wk_dataset" / "datasource-properties.json"
     ) as datasource_properties:
         data = json.load(datasource_properties)
-        assert data["dataLayers"][0].get("num_channels") == 1
+        assert data["dataLayers"][0].get("numChannels") == 1
```

Comment:
The entry `num_channels` in the properties is now renamed to `numChannels` to be consistent with the other parameter. This should not be a problem for existing datasets because the parameter `num_channels` was only used by the dataset API (to my knowledge (see: https://github.com/scalableminds/webknossos/blob/master/frontend/javascripts/types/schemas/datasource.schema.js)) and it is optional.